pycolleff/pycolleff/process_wakes.py

```diff
@@ -1172,7 +1172,7 @@ def _ECHO3D_load_data(simul_data, path, anal_pl):
                 'longitudinal wake simulation.'
                 'Results might not be accurate.')
 
-        spos = mstepz*_np.arange(nz, dtype=float)
+        spos = mstepz*_np.arange(nz, dtype=float) + 0.5*mstepz

```

Comment:
I think this way is more optimized:
```
spos = _np.arange(nz, dtype=float)
spos += 0.5
spos *= mstepz
```

Reply:
Thank you @fernandohds564, I made a new commit with the suggested changes.